src/potree.ts

```diff
@@ -47,12 +49,16 @@ export class Potree implements IPotree {
   features = FEATURES;
   lru = new LRU(this._pointBudget);
 
-  loadPointCloud(
-    url: string,
-    getUrl: GetUrlFn,
-    xhrRequest = (input: RequestInfo, init?: RequestInit) => fetch(input, init),
-  ): Promise<PointCloudOctree> {
-    return loadPOC(url, getUrl, xhrRequest).then(geometry => new PointCloudOctree(this, geometry));
+  async loadPointCloud(url: string, getUrl: GetUrlFn, xhrRequest = (input: RequestInfo, init?: RequestInit) => fetch(input, init)): Promise<PointCloudOctree> {
+    if (url === 'cloud.js') {
+      return await loadPOC(url, getUrl, xhrRequest).then((geometry) => {
+        return new PointCloudOctree(this, geometry);
+      });
+    } else if (url === 'metadata.json') {
+      return await loadOctree(url, getUrl, xhrRequest).then((geometry: OctreeGeometry) => {
+        return new PointCloudOctree(this, geometry); });
+    }
+    throw new Error('Unsupported file type');
```

Comment:
Thanks. I like this approach because it allows us to leave our existing loader untouched and simply pass the v2 for the new output type. I suspect there will be yet another type here eventually which would be the loader for the glTF data layout.